index.html

```diff
@@ -45,7 +45,7 @@ <h2 class="featurette-heading">What is Go Ethereum?</h2>
         <div class="col-md-7">
           <h2 class="featurette-heading">How can I use it?</h2>
           <p class="lead">Go Ethereum is available either as a standalone client called Geth that you can install on pretty much any operating system, or as a library that you can embed in your Go, Android or iOS projects.</p>
-          <p class="lead">See our <a href="./install">installation guide</a> or our <a href="https://github.com/ethereum/go-ethereum/wiki" target="_blank">wiki pages</a> for details!</p>
+          <p class="lead">See our <a href="./docs/install-and-build/installing-geth">installation guide</a> or our <a href="./docs/" target="_blank">wiki pages</a> for details!</p>
```

Comment:
maybe we should not call it "wiki pages" then anymore when the target is no wiki - or remove that part all together